tests/strictequal.test.js

```diff
@@ -0,0 +1,52 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictequal')
+
+describe('test 1 fail on parameter types types', () => {
+  describe('test 1 parameter are different types', () => {
+    it('test 1 types differ', () => {
+      // create test data for fail test
+      const firstValTest1 = '5'
+      const secondValTest1 = 5
+
+      //call the function
+      let test1Result = strictEqual(firstValTest1, secondValTest1)
```

Comment:
Since this data is so simple, you can one line all of this rather than creating extra variables.

```JavaScript
const result = strictEqual('5', 5)
```